src/renderer/components/sidebar-file-tree.tsx

```diff
@@ -160,7 +160,7 @@ export const SidebarFileTree = observer(
       ];
 
       return (
-        <div style={{ overflow: 'hidden' }}>
+        <div style={{ overflow: 'auto' }}>
```

Comment:
```suggestion
        <div style={{classNames('fiddle-scrollbar', isWin && 'win')}}>
```

```css
.fiddle-scrollbar {
  overflow: auto;
}

.win.fiddle-scrollbar::-webkit-scrollbar {
  width: 8px;
}

.win.fiddle-scrollbar::-webkit-scrollbar-thumb {
  border-radius: 4px;
  background-color: rgba(172, 172, 172, 0.4);
}

.win.fiddle-scrollbar::-webkit-scrollbar-track {
  background-color: transparent;
}
```


https://user-images.githubusercontent.com/8198408/196402225-2be47ee5-7d84-4aa8-a9c9-27c696154534.mp4

Reply:
Ah nice! Did a bit of testing and I think the `win` prefix may not be needed as the styling also works on MacOS (and I presume Linux)

I've also [found an existing pattern](https://github.com/electron/fiddle/blob/9edd8f2852c1237aeed379e33a9ae9abc018de86/src/less/components/sidebar.less#L49-L51) to slightly change the scrollbar based on the theme as it seemed a contrast on the light theme was not good

Changes updated and preview below
<img width="956" alt="image" src="https://user-images.githubusercontent.com/892961/197338944-118ae4b3-ba6a-4d47-8c5e-8d2679d957ef.png">

Things are still running through checks and will try address if anything fails